src/nni_manager/core/test/nnimanager.test.ts

```diff
@@ -71,29 +73,71 @@ describe('Unit test for nnimanager', function () {
         }
     }
 
+    let updateExperimentParams = {
+        authorName: '',
+        experimentName: 'another_experiment',
+        trialConcurrency: 3,
+        maxExecDuration: 6,
+        maxTrialNum: 2,
+        trainingServicePlatform: 'local',
+        searchSpace: '{"y":2}',
+        tuner: {
+            className: 'TPE',
+            classArgs: {
+                optimize_mode: 'maximize'
+            },
+            checkpointDir: '',
+            gpuNum: 0
+        },
+        assessor: {
+            className: 'Medianstop',
+            checkpointDir: '',
+            gpuNum: 1
+        }
+    }
+
+    let experimentProfile = {
+        params: updateExperimentParams,
+        id: 'test',
+        execDuration: 0,
+        maxSequenceId: 0,
+        revision: 0
+    }
+    
+
     before(async () => {
         await initContainer();
         nniManager = component.get(Manager);
         const expId: string = await nniManager.startExperiment(experimentParams);
-        assert(expId);
-    });
+        assert.strictEqual(expId, 'unittest');
+    })
 
     after(async () => {
         await nniManager.stopExperiment();
         cleanupUnitTest();
     })
 
-    it('test resumeExperiment', () => {
-        //TODO: add resume experiment unit test
+
+
+    it('test addCustomizedTrialJob', () => {
+        return nniManager.addCustomizedTrialJob('hyperParams').then(() => {
+
+        }).catch((error) => {
+            assert.fail(error);
+        })
     })
 
+
     it('test listTrialJobs', () => {
-        //FIXME: not implemented
-        //return nniManager.listTrialJobs().then(function (trialJobDetails) {
-        //    expect(trialJobDetails.length).to.be.equal(2);
-        //}).catch(function (error) {
-        //    assert.fail(error);
-        //})
+        return nniManager.listTrialJobs().then(function (trialjobdetails) {
+            expect(trialjobdetails.length).to.be.equal(2);
+            expect(trialjobdetails[0].id).to.be.equal('4321'); // generated by init
+            expect(trialjobdetails[0].status).to.be.equal('WAITING');
+            expect(trialjobdetails[1].id).to.be.equal(''); // generated by addCustomizedTrialJob
```

Comment:
listTrialJobs methods does not guarantee order of jobs,  the first expect is enough for nnimanager UT.  The remaining 4 expects can be removed, they should be tested by datastore UT.